src/Services/PluginService.cs

```diff
@@ -160,7 +160,14 @@ public async Task<IEnumerable<IPluginWrapper>> GetInstalledPluginsAsync(bool inc
                 }
 
                 var name = extension.DisplayName;
-                var pluginWrapper = new PluginWrapper(name, extension.Package.Id.FullName, classId);
+                var pluginWrapper = new PluginWrapper(

```

Comment:
Discussed offline, but this current mapping of extension to package is flawed and doesn't handle all possible combinations. Fixing this issue for the plugin is beyond the scope of this PR and that's a large can of worms in its own right, but what you can do is take one step towards correctness by adding a few more bits of data and then using a safe Unique Extension Identifier. 

It is good to add these properties to the plugin wrapper, but I recommend adding one more which acts as a unique identifier of the extension.  To get the unique identifier, use two other data points in the extension object:

1) The AUMID of the application of the extension. This is in the extension object, as the "AppUserModelId".  The AUMID is the concatenation of the package family name and the application id and uniquely identifies the application containing the extension within the package.
https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.appextensions.appextension.appusermodelid?view=winrt-22621

2) The Extension Id. This is the unique identifier of the extension within the application.
https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.appextensions.appextension.id?view=winrt-22621

Add the composite of the above two as the unique identifier of the extension across all dev home extensions:

var extensionUniqueIdentifier = extension.AppUserModelId + '!' + extension.Id;

Add this to the plugin wrapper, and use this for the setting name for enabling/disabling an extension. This ensures that in the future the setting will remain valid even after we fix the rest of this to properly handle multiple applications in a package or multiple extensions in an application.